### DISCUSSION TITLE
How to find tableId

### DISCUSSION THREAD

**Discussion by praajna-shettigar** [Budibase/budibase#8481] [Help] [answered]
Hi,

How to find tableId of particular table to use it in API reference

Thanks,

**Comment by andz-bb** [comment, marked as answer]
Hey @praajna-shettigar - your table ID can be found in the URL when you're viewing that table in the data section of the builder. The ID will be after `/table/` in the URL - mine is highlighted below:

<img width="1308" alt="Screenshot 2022-11-02 at 10 49 07" src="https://user-images.githubusercontent.com/110921612/199470972-05d2ef91-6422-4b55-b203-16ca6cd82759.png">

**Reply by melohagan** [reply]
You can also use the API to search for tables by name, which will return the ID of the table in the response: https://docs.budibase.com/reference/post_tables-search
You would need the app ID to do that, which is also found in the URL after `/app/`
